### PR TITLE
TX Current 2019 reports for both 28 days and 90 days

### PR DESCRIPTION
**Pull request by dbaluku**
https://app.asana.com/0/1133167201254913/1153407010912289/f

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/ugandaemrreports/reports/SetupTxCurrent_28Days2019Report.java [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+    }
+
+    @Override
+    public String getName() {
+        return "Tx_Current28Days 2019 Report";
+    }
+
+    @Override
+    public String getDescription() {
+        return "MER Indicator Report For Tx Curr with Lost To Followup taken as 28 Days After last Missed Appointment";
```
Comment (ssmusoke) [inline api/src/main/java/org/openmrs/module/ugandaemrreports/reports/SetupTxCurrent_28Days2019Report.java]: Change description to "TX_CURR MER Indicator report for PEPFAR with Lost to Followup taken as 28 days from the last scheduled appointment"

File: api/src/main/java/org/openmrs/module/ugandaemrreports/reports/SetupTxCurrent_90Days2019Report.java [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+    }
+
+    @Override
+    public String getName() {
+        return "Tx_Current90Days 2019 Report";
+    }
+
+    @Override
+    public String getDescription() {
+        return "MER Indicator Report For Tx Curr with Lost To Followup taken as 90 Days After last Missed Appointment";
```
Comment (ssmusoke) [inline api/src/main/java/org/openmrs/module/ugandaemrreports/reports/SetupTxCurrent_90Days2019Report.java]: Change description to "TX_CURR MER Indicator report for MoH with Lost to Followup taken as 90 days from the last scheduled appointment"

File: api/src/main/java/org/openmrs/module/ugandaemrreports/reports/SetupTxCurrent_28Days2019Report.java (hunk 1 above)
Comment (dbaluku) [reply api/src/main/java/org/openmrs/module/ugandaemrreports/reports/SetupTxCurrent_28Days2019Report.java]: Done

File: api/src/main/java/org/openmrs/module/ugandaemrreports/reports/SetupTxCurrent_90Days2019Report.java (hunk 2 above)
Comment (dbaluku) [reply api/src/main/java/org/openmrs/module/ugandaemrreports/reports/SetupTxCurrent_90Days2019Report.java]: Done

**Pull request merged**